### PR TITLE
Core/1 Mailgun Integration

### PR DESCRIPTION
**Pull request by MauricioDinki**

### CODE REVIEW

**Comment by Enrique199413** [comment]
Is necesary add the guide with steps for configure integration mailgun? @MauricioDinki

**Comment by MauricioDinki** [comment]
Yes, in the wiki we add the configuration steps

**Comment by Enrique199413** [comment]
Use same PR #2  or add another?

**Comment by MauricioDinki** [comment]
We add the docs to both, at the end, after the mailchimp integration

File: integrations/templates/email/generic_contact.html [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Nombre del cliente: {{ name }}
+Correo electronico: {{ email }}
```
Comment (jarodriguez80) [inline integrations/templates/email/generic_contact.html]: Must be electrónico

**Pull request merged**